Purge "will" from Client Cfg (#663)
uyuni-project/uyuni-docs: modules/client-configuration/pages/vhm-caasp.adoc

[NOTE]
====

> You can not use the management node until the highstate has been completed.

Reviewer: ```suggestion
You cannot use the management node until the highstate has been completed.
```
